frontend/components/common.js

```diff
@@ -2,6 +2,13 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import styled, { css } from 'styled-components'
 
+const noAutocorrect = {
```

Comment:
Nit: Maybe do `Object.freeze` on this to ensure it's not mutated anywhere? 

If the repo were using strong typing, it'd be fine (eg. in Flow you can annotate an object as `$ReadOnly` and then Flow will throw a type error if you try to mutate it.